jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenterTest.java

```diff
@@ -362,6 +365,35 @@ public void testDefaultAdvancedSearchFilterTypes() {
                                          any(Consumer.class));
     }
 
+    @Test
+    public void testOnSelectionEvent() {
+        String deploymentId = "evaluation.1.0.1";
+        String errorId = "testErrorId";
+        String processId = "Evaluation";
+        Long processInstanceId = Long.valueOf(1);
+        ExecutionErrorSummary errorSummary =
+                ExecutionErrorSummary.builder()
+                        .errorId(errorId)
+                        .deploymentId(deploymentId)
+                        .processId(processId)
+                        .processInstanceId(processInstanceId)
+                        .build();
+        when(placeManager.getStatus(any(DefaultPlaceRequest.class))).thenReturn(PlaceStatus.CLOSE);
+        presenter.setExecErrorSelectionEvent(execErrorSelectionEventMock);
+        presenter.selectExecutionError(errorSummary,
+                                       true);
+        final ArgumentCaptor<ExecErrorSelectionEvent> captor = ArgumentCaptor.forClass(ExecErrorSelectionEvent.class);
+        verify(execErrorSelectionEventMock).fire(captor.capture());
+        assertEquals(deploymentId,
+                     captor.getValue().getDeploymentId());
+        assertEquals(errorId,
+                     captor.getValue().getErrorId());
+        assertEquals(errorSummary.getProcessId() +
+                             " - " + errorSummary.getProcessInstanceId() +
+                             " (" + errorSummary.getDeploymentId() + ")",
+                     captor.getValue().getErrorDetailsTitle());
+    }
+
```

Comment:
@nmirasch Can we add one test which asserts all the columns which should be shown by default?